src/sagas/wallet.js

```diff
@@ -188,6 +189,12 @@ export function* startWallet(action) {
   // Thread to listen for feature flags from Unleash
   const featureFlagsThread = yield fork(listenForFeatureFlags, featureFlags);
 
+  const threads = [
```

Comment:
Maybe add some comments here?

Reply:
Done in 0b85cd1261820046ed98aac488e2326fe2fd611d